ruby_on_rails/rspec.md

```diff
@@ -58,30 +57,24 @@ Kernel.srand config.seed
 
 Please check the [spec_helper template](../templates/spec/spec_helper.rb)
 
-* Inside `spec/rails_helper` we suggest to uncomment/enable the following:
-
 * after `require 'rspec/rails'`
 
 ```ruby
 require 'capybara/rspec'
 require 'capybara/rails'
 require 'selenium/webdriver'
 
-Capybara.register_driver :chrome do |app|
-  Capybara::Selenium::Driver.new(app, browser: :chrome)
-end
+RSpec.configure do |config|
+ # other configs
```

Comment:
is this comment needed?

Reply:
and is `config.use_transactional_fixtures = true` the default?

Reply:
right! I added it to the rails_helper.rb template

Reply:
for the comment: I wanted to show that the previous configurations are there, but is not in the template